mp/src/game/client/momentum/ui/controls/UserComponent.cpp

```diff
@@ -86,8 +83,6 @@ void UserComponent::OnUserDataUpdate()
 
 void UserComponent::FillControlsWithUserData(const UserData &userData)
 {
-    m_pUserName->SetText(userData.m_szAlias);
```

Comment:
I do not agree with removing this, this is data cached out in the user data file. Would be good to have this still as it sets a name before it catches up with Steam API stuff.